### DISCUSSION TITLE
You nailed it!

### DISCUSSION THREAD

**Discussion by sideral** [mantinedev/mantine#6150] [Kind words]
Man, I've been doing Front-End for 20 years. This is, hands-down, the best component library I've ever used. What's more, the parts that I didn't like (Styling from JS Objects, slow with big forms) have been addressed in v7. Please keep it up, this library deserves more exposure, it just works and works well and beautifully. Many thanks to @rtivital  and the contributors!

**Comment by rtivital** [comment]
Thanks for kind words!
